src/ccs/multifolding.rs

```diff
@@ -76,8 +76,8 @@ impl<G: Group> Multifolding<G> {
   ) -> G::Scalar {
     let mut c = G::Scalar::ZERO;
 
-    let e1 = eq_eval(r_x, r_x_prime);
-    let e2 = eq_eval(beta, r_x_prime);
+    let e1 = EqPolynomial::new(r_x.to_vec()).evaluate(r_x_prime);
+    let e2 = EqPolynomial::new(beta.to_vec()).evaluate(r_x_prime);
```

Comment:
The `to_vec` forces the clone. 
Can't we modify the fn signature then so that forces us to clone too? (So no reference)?? Otherwise, this looks like you just need a ref while in reality, is basically copying the entire vector inside.

Reply:
Hm, I'd rather not change Nova code API (`EqPolynomial::new` args) in this PR tbh unless strong need, since it touches upstream and other stuff